uamqp/async_ops/client_async.py

```diff
@@ -424,6 +424,11 @@ class SendClientAsync(client.SendClient, AMQPClientAsync):
      will assume successful receipt of the message and clear it from the queue. The
      default is `PeekLock`.
     :type receive_settle_mode: ~uamqp.constants.ReceiverSettleMode
+    :param desired_capabilities: The extension capabilities desired from the peer endpoint.
+     To create a desired_capabilities object, please do as follows:
+        - 1. Create an array of desired capability symbols: `capabilities_symbol_array = [types.AMQPSymbol(string)]`
+        - 2. Transform the array to AMQPValue object: `utils.data_factory(types.AMQPArray(capabilities_symbol_array))`
```

Comment:
Not blocking right now - but could we not automatically perform this transformation if someone passes in a list of strings?
So type could be Union[AMQPValue, List[str]]

Reply:
yea, I think that should be doable and would be nice to have.

I have created an separate issue for this one: https://github.com/Azure/azure-uamqp-python/issues/212